onesixtyone.c

```diff
@@ -142,7 +142,11 @@ void read_communities(char* filename)
     printf("MAX_COMMUNITIES (%d) reached. Remaining communities will be skipped \n", i);
   }
 
-  community_count = i;
+  if (c == 0 && ch == -1)
+	community_count = i - 1;
+  else
+	community_count = i;
```

Comment:
One last nit: mind using spaces for indentation here, for consistency with the rest of the file?

Reply:
I really don't mind hahahaha

Done in 7584510b91a389b15b220784e4eaf078a3b905a1

Reply:
Thanks!